include/exadg/incompressible_navier_stokes/time_integration/time_int_bdf_dual_splitting.cpp

```diff
@@ -712,6 +712,24 @@ TimeIntBDFDualSplitting<dim, Number>::viscous_step()
       velocity_np = velocity_viscous_last_iter;
     }
 
+    /*
+     *  update variable viscosity
+     */
+    if(this->param.viscous_problem() and this->param.viscosity_is_variable() and
+       this->param.treatment_of_variable_viscosity == TreatmentOfVariableViscosity::Explicit)
+    {
+      dealii::Timer timer_viscosity_update;
+      timer_viscosity_update.restart();
```

Comment:
This line is not necessary (and was not in the code where this was moved from), because the constructor already starts the timer,
```suggestion
```

Reply:
Thanks for this hint! We might have this line quite often in the code, so it could be best to address this separately.